### PR TITLE
update automating backups script

### PR DESCRIPTION
**Pull request by Mannuel25**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
All the backslashes, except the one in the last line, are required. The tell the shell that the command is continued on the next line.

Also the `--glob-archives` need to match the hostname as this is used in the `borg create` command.

You can just copy and paste the script after you changed it and practically try it out to find any issues.

**Comment by ThomasWaldmann** [comment]
Also, `-a` is the short alias of `--glob-archives` - one just decides for one of them. For an example script, using the long form is usually better readable.

**Comment by Mannuel25** [comment]
Okay, thanks for the corrections.

File: docs/quickstart.rst [hunk 1]
```diff
@@ -202,13 +202,13 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
...
-    borg prune                          \
-        --list                          \
-        --prefix '{hostname}-'          \
-        --show-rc                       \
-        --keep-daily    7               \
-        --keep-weekly   4               \
-        --keep-monthly  6               \
+    borg prune                      \
+    --list                          \
+    --glob-archives '{hostname}-'   \
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst:207]: needs to be `--glob-archives '{hostname}-*'`.

also maybe rather keep the indentation of the lines with the options, so it is better to see that they belong to the first line with the beginning of the command.

In line 211, remove all spaces before the line end, so there are no trailing spaces.

also, fix the create command in the same way.

Comment (Mannuel25) [reply docs/quickstart.rst:207]: Okay, I'll fix it. But I didn't get you here: `keep the indentation of the lines with the options, so it is better to see that they belong to the first line with the beginning of the command.`

Comment (Mannuel25) [reply docs/quickstart.rst:207]: Okay, I'll fix it. Thanks

**Review by ThomasWaldmann — APPROVED** [review]
Looks good now!

**Comment by Mannuel25** [comment]
> Looks good now!

Thanks for helping out 😁

**Comment by ThomasWaldmann** [comment]
Merged, thanks!

**Comment by Mannuel25** [comment]
> Merged, thanks!

Thank you!

**Pull request merged**